http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/DataprocRuntimeMonitor.scala

```diff
@@ -187,7 +187,9 @@ class DataprocRuntimeMonitor[F[_]: Parallel](
                         failedRuntime(
                           monitorContext,
                           runtimeAndRuntimeConfig,
-                          RuntimeErrorDetails(s"Can't find master instance for this cluster"),
+                          RuntimeErrorDetails(s"Can't find master instance for this cluster",
```

Comment:
Would you mind adding documentation to the `RuntimeErrorDetails` object so people understand what the parameters are used for?